Classes/Domain/AbstractAggregateRoot.php

```diff
@@ -110,7 +104,7 @@ protected function __construct(string $identifier)
      */
     final protected function apply(EventInterface $event)
     {
-        $method = sprintf('when%s', $this->eventTypeService->getEventShortType($event));
+        $method = sprintf('when%s', (new \ReflectionClass($event))->getShortName());
```

Comment:
True. There are some more naming oddities and CGL violations. I suggest to address them all in a "cosmetic fix"